common/lib/xmodule/xmodule/contentstore/mongo.py

```diff
@@ -141,6 +144,9 @@ def find(self, location, throw_on_not_found=True, as_stream=False):
                 )
             else:
                 with self.fs.get(content_id) as fp:
+                    # Need to replace dict IDs with SON for chunk lookup to work under Python 3
```

Comment:
Can you add a second comment line `# because item order can be different and mongo cares about the item order.`